terminator/lib/fwss.js

```diff
@@ -0,0 +1,15 @@
+export const abi = [
+  {
+    type: 'function',
+    name: 'terminateCDNService',
```

Comment:
In the past, we were using Solidity ABI syntax:
- https://github.com/filcdn/bot/blob/71414382d34437644a24b34f48135aebe3b176b3/index.js#L14-L39
- https://github.com/filcdn/worker/blob/1b8aed7cd7a5bae13b4234557c21c492faf172d7/indexer/lib/pdp-verifier.js#L5-L8 (we removed this as part of M2.5 upgrade)

I don't mind switching to JSON ABI as you do here, but I'd like to avoid inconsistency. I'd like us to either use JSON ABI everywhere, or use Solidity ABI everywhere, but not both.

Reply:
Since we're using viem here, maybe it's different?

Reply:
Fair enough :+1:

If we are using Viem in our workers, should be eventually migrate the bot from Ethers to Viem too? (We can create a tech-debt issue in the bot repo, and ask PLDG to contribute.)